Formula/ehco.rb

```diff
@@ -5,6 +5,11 @@ class Ehco < Formula
   sha256 "326c468c3790ad01031e52ccb4efcfa5e331d2198fdb13137749c67e8eaacf38"
   license "GPL-3.0-only"
 
+  livecheck do
+    url :stable
+    strategy :github_latest
```

Comment:
We should tighten the regex used instead of using `:github_latest`.

Reply:
yeah, I tried the option of using the regular regex, but it did not quite work out.

Reply:
Assuming they don't add another dotted section, I guess you can restrict the number of sections:
```suggestion
    regex(/^v?(\d+(?:\.\d+){,2})$/i)
```

Reply:
Or maybe:
```suggestion
    regex(/^v?(\d+(?:\.\d+){1,2})$/i)
```
since we prefer at least one dot?